File: MPI/mpicheck/tests/functions/4.0/MPI_Intercomm_create_from_groups.f90

        
        program main
        use mpi
        
        INTEGER var_0
       INTEGER var_1
       INTEGER var_2
       INTEGER var_3
       CHARACTER*(10) var_4
       INTEGER var_5
       INTEGER var_6
       INTEGER var_7
       INTEGER var_8
        call mpi_intercomm_create_from_groups(var_0, var_1, var_2, var_3, var_4, var_5, var_6, &
       var_7, var_8)
       call pmpi_intercomm_create_from_groups(var_0, var_1, var_2, var_3, var_4, var_5, var_6, &
       var_7, var_8)
        end program main
    

File: MPI/mpicheck/tests/functions/4.0/MPI_Intercomm_create_from_groups.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Group var_0;
    int var_1;
    MPI_Group var_2;
    int var_3;
    const char *var_4;
    MPI_Info var_5;
    MPI_Errhandler var_6;
    MPI_Comm *var_7;
    int ret;
    /* calls */
    ret = MPI_Intercomm_create_from_groups(var_0, var_1, var_2, var_3, var_4, var_5, var_6, var_7);
    ret = PMPI_Intercomm_create_from_groups(var_0, var_1, var_2, var_3, var_4, var_5, var_6, var_7);
    return 0;
}
